conans/paths/package_layouts/package_editable_layout.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+
+import os
+
+from conans.model.ref import ConanFileReference
+from conans.model.ref import PackageReference
+from conans.paths import CONANFILE
+from conans.util.files import load
+
+CONAN_PACKAGE_LAYOUT_FILE = '.conan_package_layout'
```

Comment:
The cache file is named ``"editable_packages.ini"``. I would love to see some symmetry of these 2 files. They have very similar syntax and are loaded and parsed with the same class, a similar filename would be more intuitive for users.